### PR TITLE
Fix ic_volume_up and ic_volume_off being black

### PR DESCRIPTION
**Pull request by mhmdanas**
#### What is it?
- [x] Bugfix (user facing)

#### Description of the changes in your PR

(Possibly) fix an issue where ic_volume_up is shown in black.

I could not reproduce the issue, so I am unable to confirm if my changes actually fix the problem. @AlexN1997 can you see if this PR fixes the problem?

#### Fixes the following issue(s)

- https://github.com/TeamNewPipe/NewPipe/pull/6037#issuecomment-817137907

#### APK testing 

On the website the APK can be found by going to the "Checks" tab below the title and then on "artifacts" on the right.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by AlexN1997** [comment]
Hi @mhmdanas,
I tested your apk with your PR and it works!

Thanks alot

**Review by TiA4f8R — APPROVED** [review]

**Comment by TobiGr** [comment]
Can you also take a look at https://github.com/TeamNewPipe/NewPipe/issues/6052#issuecomment-818181125

**Comment by mhmdanas** [comment]
@TobiGr yes, I also noticed that issue, but that icon already has `android:tint` set to white, so I have no idea what the cause is.

**Comment by mhmdanas** [comment]
Please don't merge this PR yet. I am also going to fix the issue @TobiGr mentioned.

File: app/src/main/res/drawable-night/ic_volume_off.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     android:viewportWidth="24.0"
     android:viewportHeight="24.0">
     <path
-        android:fillColor="#FF000000"
+        android:fillColor="#FFFFFFFF"
```
Comment (mhmdanas) [inline app/src/main/res/drawable-night/ic_volume_off.xml:7]: This fixed the issue for me, but I have no idea if this is the correct way.

Comment (Stypox) [reply app/src/main/res/drawable-night/ic_volume_off.xml:7]: Yes, it is

**Review by Stypox — APPROVED** [review]

**Pull request merged**